internal/cluster/transition_test.go

```diff
@@ -3584,6 +3640,451 @@ var _ = Describe("Single node", func() {
 	})
 })
 
+var _ = Describe("Ocs Operator use-cases", func() {
```

Comment:
@gamli75, I have added some tests on OCS already. Let me know if anything else is required to be added. Thanks!